ArduCopter/GCS_Mavlink.cpp

```diff
@@ -991,28 +991,30 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
 
 MAV_RESULT GCS_MAVLINK_Copter::handle_command_pause_continue(const mavlink_command_int_t &packet)
 {
-    if (copter.flightmode->mode_number() != Mode::Number::AUTO) {
-        // only supported in AUTO mode
+#if defined(MODE_GUIDED_ENABLED) && defined(MODE_AUTO_ENABLED)
```

Comment:
I am not sure whether
`#if defined(MODE_GUIDED_ENABLED) && defined(MODE_AUTO_ENABLED)`
or
`#if defined(MODE_GUIDED_ENABLED) || defined(MODE_AUTO_ENABLED)`

Reply:
I think we want  `||` but I could be wrong on that.... 

Currently in `config.h` you will find the following. So we could disable auto mode but still have guided but not the other way around. (Though I'm not sure why you would disable either....)
```cpp
#if MODE_AUTO_ENABLED && !MODE_GUIDED_ENABLED
  #error ModeAuto requires ModeGuided which is disabled
#endif
```

Reply:
I think so, too. Just to be safe, I wrote as &&.